.github/workflows/pull_request.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - "master"
       - "branch-v*"
+      - "bug_fix_*"
```

Comment:
Should this be `bug-fix-`?
```suggestion
      - "bug_fix_*"
```

Reply:
I will ask this question in the session, thanks.